distributions/openhab-demo/src/main/resources/conf/services/addons.cfg

```diff
@@ -0,0 +1,39 @@
+# The installation package of this openHAB instance
+# Note: This is only regarded at the VERY FIRST START of openHAB
+# Note: If you want to specify your add-ons yourself through entries below, set the package to "minimal"
+# as otherwise your definition might be in conflict with what the installation package defines.
+#
+# Valid options:
+#   - standard : Standard setup for normal use of openHAB
+#   - minimal  : Installation of core components, but no UIs or other add-ons. Use this for special headless setups.
+#   - demo     : A pre-configured demo setup which can be used for demonstration.
```

Comment:
This PR would be a very good opportunity to remove this line :-D
Also in https://github.com/openhab/openhab-distro/blob/master/distributions/openhab/src/main/resources/conf/services/addons.cfg, of course.

Reply:
Ah yes. Good catch! I never read docs. :wink: